res_users_password_security/__openerp__.py

```diff
@@ -5,7 +5,7 @@
 
     'name': 'Password Security',
     "summary": "Allow admin to set password security requirements.",
-    'version': '9.0.1.0.1',
+    'version': '9.0.1.0.2',
```

Comment:
let's say this is not yet critical until merge. After the merge we should stick strictly to the rule